ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1048,7 +1048,10 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
       if (populationChecked) {
         values = {...values, populationDetails};
         constraints['populationDetails'] = {
-          presence: true
+          presence: true,
+          length: {
+            minimum : SpecificPopulationItems.length
```

Comment:
Should this be `1` instead of `SpecificPopulationItems.length`? The length of the array is 10 so currently you have to select 10 items to enable the the Create Workspace button.

Reply:
The 10 items are all required. We shouldn't allow user create workspace before they finish all 10 items.

Reply:
This doesn't make sense to me. `minimum : SpecificPopulationItems.length` is setting the minimum number of underrepresented population checkboxes that need to be selected. The description says:

> Yes, my study will focus on **one or more** specific underrepresented populations

So why are we requiring the user to select 10 of these options before continuing?

Reply:
![Screen Shot 2021-12-07 at 11 50 25 AM](https://user-images.githubusercontent.com/6351731/145071461-cb835a87-ed9b-42dd-b801-9b6f292c7879.png)
The 10 items are required if you choose YES.
See the screenshot above, the expected behaviour is they all required. 

RE: using of minimum. That is that I find online how to use `validate.js `to check array length (and seems that library does not offer many features for array type). But I am new to this, let me know if there is any better rules for that.

Reply:
The 10 items are age group, sex at birth, race, etc...

Reply:
Yes, I understand what you mean but I don't think all 10 sections are supposed to be required (we may want to verify with someone on this to be sure). The `*` after each item is for the footnote below the list:
![Screen Shot 2021-12-07 at 10 58 25 AM](https://user-images.githubusercontent.com/40036095/145072782-1f977aa8-3c06-4cf3-b67b-7c988b117bda.png)

Also, it doesn't check if each section has a selection, only the total number of selections, so you can select any 10 items and the button will enable:
![Screen Shot 2021-12-07 at 10 56 07 AM](https://user-images.githubusercontent.com/40036095/145073249-8bf005d4-e51b-44a9-924b-9178fd3605fa.png)

Reply:
Looks like you're adding validation to require a selection for each of the 10 sections. I just want to repeat that we should verify this is the correct UX before continuing. 

The description specifically states that the user should select "one or more" underrepresented populations, which to me means that they can create a workspace even if they only select a single checkbox.

Reply:
I think you are right, this might not be a bug but an intended behaviour. (I was playing around see how custom validation works in this library, will close the ticket and cicle back on that ticket)
Thanks!

Reply:
Just to clarify, I think the ticket is probably valid. We just need to require at least one total selection from _any_ section if they choose 'Yes' and not one selection from each section.

The error messaging for this is already in place to show in the tooltip for the create button ([line 1554](https://github.com/all-of-us/workbench/blob/master/ui/src/app/pages/workspace/workspace-edit.tsx#L1554)):

> You must specify a population of study (Question 5)

Reply:
Yes, I think that is probably right, and I am checking back with Shimon on that via Jira

Reply:
it should be "at lease 1 be selected". So min length = 1 is the right validation we should add.
I can not reopen the pull request because pf defaul branch change, will open a new one. 
Thanks! @dolbeew